.github/workflows/linux-build-and-test.yaml

```diff
@@ -194,8 +197,13 @@ jobs:
       if: always()
       id: collectlogs
       run: |
+        set -x
         find . -name regression.diffs -exec cat {} + > regression.log
         find . -name postmaster.log -exec cat {} + > postgres.log
+        find . -name postmaster.json
+        find . -name postmaster*
```

Comment:
I guess these three lines are only for debugging and will be removed in the final PR. Or are they used for something?

Reply:
Yes, I'm going to remove it. The json logs are not generated for some reason...